### PR TITLE
refactor, chore(egen): xgboost package version set to 1.7.1, updates serving container image to 1.7, deletes intermediate files, other fixes from template

### PR DESCRIPTION
**Pull request by SumanthKasula99**
This PR consists of the following updates:

1. XGBoost version set to 1.7.1
2. Updates serving container image to xgboost-cpu.1-7:latest.
3. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
4. Adds link for Colab enterprise.
5. Cleans up the intermediates files created.
6. Sets location to "us-central1".

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb [hunk 1]
```diff
@@ -380,7 +350,7 @@
       "source": [
         "#### UUID\n",
         "\n",
-        "If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a uuid for each instance session, and append it onto the name of resources you create in this tutorial."
+        "To avoid name collisions between users on created resources, create a uuid for each session instance. Append these uuids to the respective names of the resources created in this tutorial."
```
Comment (kittyabs) [inline notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:353]: - change: "uuids"
- to: "UUIDs"

**Review by kittyabs — COMMENTED** [review]
small edit

**Review by katiemn — APPROVED** [review]

**Pull request merged**